apps/native-component-list/src/screens/Image/tests/appearance.tsx

```diff
@@ -72,6 +72,7 @@ const imageTests: ImageTestGroup = {
     {
       name: 'Tint color',
       props: {
+        tintColor,
```

Comment:
Similarly to Android, this prop in the original image is actually passed as a style that we don't propagate to the props. This is a temporary workaround for tint color, but to be honest I think we should deprecate using it as a style.